packages/manager/src/features/PlacementGroups/PlacementGroupsAssignLinodesDrawer.tsx

```diff
@@ -0,0 +1,256 @@
+  );
+  const linodeSelectLabel = placementGroupRegion
+    ? `Linodes in ${placementGroupRegion.label} (${placementGroupRegion.id})`
+    : 'Linodes';
+
+  const drawerTitle =
+    label && affinity_type
+      ? `Add Linodes to Placement Group ${label} (${AFFINITY_TYPES[affinity_type]})`
+      : 'Add Linodes to Placement Group';
+
+  const removableSelectionsListLabel = (
+    <>
+      <FormLabel htmlFor="pg-linode-removable-list">
+        {label && affinity_type
+          ? `Linodes Assigned to Placement Group ${label}
+        (${AFFINITY_TYPES[affinity_type]})`
+          : 'Linodes Assigned to Placement Group'}
+      </FormLabel>
+      <Typography component="span" display="block" fontSize="0.8rem">
+        Maximum Number of Linodes for this group:{' '}
+        {selectedPlacementGroup.capacity}
+      </Typography>
+    </>
+  );
+
+  const handleAssignLinode = async (e: React.SyntheticEvent<HTMLElement>) => {
+    e.preventDefault();
+    setSelectedLinode(null);
+
+    if (!selectedLinode) {
+      return;
+    }
+
+    setLocalLinodesSelection([...localLinodesSelection, selectedLinode]);
+
+    const payload: AssignLinodesToPlacementGroupPayload = {
+      linodes: [selectedLinode.id],
+    };
+
+    try {
+      await assignLinodes(payload);
+    } catch (error) {
+      setGeneralError(
+        error?.[0]?.reason
+          ? error[0].reason
+          : 'An error occurred while adding the Linode to the group'
+      );
+    }
+  };
+
+  const handleUnassignLinode = (linode: Linode) => {
+    setLocalLinodesSelection(
+      localLinodesSelection.filter((l) => l.id !== linode.id)
+    );
+
+    const payload: UnassignLinodesFromPlacementGroupPayload = {
+      linodes: [linode.id],
+    };
+
+    try {
+      unassignLinodes(payload);
+    } catch (error) {
+      setGeneralError(
+        error
+          ? error?.[0]?.reason
+          : 'An error occurred while removing the Linode from the group'
+      );
+    }
+  };
+
+  const placementGroupFull = hasPlacementGroupReachedCapacity(
+    selectedPlacementGroup
+  );
+
+  return (
+    <Drawer onClose={onClose} open={open} title={drawerTitle}>
+      <Grid>
+        {generalError ? <Notice text={generalError} variant="error" /> : null}
+        <form onSubmit={handleAssignLinode}>
+          <Stack spacing={1}>
+            <Typography>
+              A Linode can only be assigned to a single Placement Group.
+            </Typography>
+
+            <Typography>
+              If you need to add a new Linode, go to{' '}
+              <Link to="/linodes/create">Create Linode</Link> and return to this
+              page to assign it to this Placement Group.
+            </Typography>
+            <LinodeSelect
+              onSelectionChange={(value) => {
+                setSelectedLinode(value);
+              }}
+              disabled={placementGroupFull}
+              helperText="Only displaying Linodes that aren’t assigned to a Placement Group"
```

Comment:
Thoughts on putting this next to the label in a tooltip?
![image](https://github.com/linode/manager/assets/115299789/c9f8c2e8-6dc8-4353-b64e-3b864a252b5b)

```suggestion
              helperText="Only Linodes that aren’t assigned to a Placement Group are displayed"
```

Reply:
Good suggestion - I'll bring this to UX but will leave it as is for this PR. Couple reasons:
- We don't have that capability for the LinodeSelect yet, so it would have to be added in a dedicated PR if we wanna go that route (i think it would be some work to do a render prop for the label
- The copy isn't final anywhere yet and we'll do a global sweep when it is